### PR TITLE
Xenobiology Map Overhaul

### PR DESCRIPTION
**Pull request by BurgerLUA**
# Overview
Revamps Xenobiology so it's worthy of a research station.

## Notable changes:
- Every cell has its own gas chamber and pipe.
- 8 cells instead of 6.
- Each cell has its own APC.
- Each cell is labeled appropriately for easy reference.
- Additional slime surgery room.
- 2 canisters of cold oxygen, and one canister of knockout gas for containment purposes.
- 3 Cardox grenades, just in case of slime outbreak.
- Stun batons now have a 95% chance to pacify slimes, 5% chance to cause them to go "rabid"
- 2 Slime Batons (weaker stunbatons) are placed in xenobiology.

Feedback thread here: https://forums.aurorastation.org/viewtopic.php?f=18&t=11580

### CODE REVIEW

**Comment by BurgerLUA** [comment]
since travis won't eat my ass, each cell will no longer have it's own apc

**Comment by XanderDox** [comment]
slime batons will stun slimes?

**Comment by BurgerLUA** [comment]
Slime Batons will pacify them, however there is a 5% chance for it to become berserk.

**Comment by Pacmandevil** [comment]
what else would a slime baton do

**Comment by Arrow768** [comment]
Afaik cardox is a expensive and highly specialized chem lore-wise.
I do not like the idea of handing them out.
Currently the only cardox grenades are inside of the vault and one in the CEs office.

In addition, I believe that there is insufficient camera coverage of xenobio.
Please ensure all non-maint areas are covered.

**Comment by skull132** [comment]
Conflicted with your own additions. Should be good after that, me thinks.

**Comment by BurgerLUA** [comment]
@skull132 merge me daddy

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -1116,21 +1116,7 @@
 	. = ..()
 	reagents.add_reagent("nutriment", 3)
 
-/obj/item/weapon/reagent_containers/food/snacks/monkeyburger/burger
```
Comment (Arrow768) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Is the removal of that one intentional ?

**Pull request merged**